src/main.cpp

```diff
@@ -13,36 +13,39 @@ int main(int argc, char **argv) {
 
   UserArgumentStruct UserArguments;
   std::string Broker;
+  App.add_flag("-C, --consumer", UserArguments.ConsumerMode,
+               "Run the program in the consumer mode.");
+  App.add_flag("-L, --list", UserArguments.MetadataMode,
+               "Metadata mode. Show all topics and partitions. If \"-t\" "
+               "specified, shows partition offsets.");
+  App.add_option("-b,--broker", Broker, "Hostname or IP of Kafka broker.");
+  App.add_option("-t, --topic", UserArguments.Name,
+                 "Show records of specified topic.");
+  App.add_option("-p,--partition", UserArguments.PartitionToConsume,
+                 "Partition to get messages from.");
   App.add_option(
          "-g, --go", UserArguments.GoBack,
          "How many records back to show from partition \"-p\". Mutually "
-         "exclusive with \"--Offset\"")
+         "exclusive with \"--Offset\".")
       ->check(CLI::Range(int64_t(0), std::numeric_limits<int64_t>::max()));
-  App.add_option("-t, --topic", UserArguments.Name,
-                 "Show records of specified topic");
-  App.add_option("-b,--broker", Broker, "Hostname or IP of Kafka broker");
+
   App.add_option("-o,--offset", UserArguments.OffsetToStart,
                  "Start consuming from an offset. Otherwise print entire "
-                 "topic. Mutually exclusive with \"--go\"")
+                 "topic. Mutually exclusive with \"--go\".")
       ->check(CLI::Range(int64_t(0), std::numeric_limits<int64_t>::max()));
-  App.add_option("-p,--partition", UserArguments.PartitionToConsume,
-                 "Partition to get messages from");
+
   App.add_option(
          "-i,--indentation", UserArguments.Indentation,
          "Number of spaces used as indentation. Range 0 - 20. 4 by default.")
       ->check(CLI::Range(0, 20));
 
   App.add_flag("-a, --all", UserArguments.ShowAllTopics,
                "Show a list of topics. To be used in \"-L\" mode.");
-  App.add_flag("-C, --consumer", UserArguments.ConsumerMode,
-               "Run the program in the consumer mode");
-  App.add_flag("-L, --list", UserArguments.MetadataMode,
-               "Metadata mode. Show all topics and partitions. If \"-t\" "
-               "specified, shows partition offsets.");
-  App.add_flag("-E, --entire", UserArguments.ShowEntireMessage,
-               "Show all records of a message(truncated by default)");
 
-  App.set_config("-c,--config_file", "", "Read configuration from an ini file",
+  App.add_flag("-e, --entire", UserArguments.ShowEntireMessage,
+               "Show all records of a message(truncated by default).");
+
+  App.set_config("-c,--config_file", "", "Read configuration from an ini file.",
```

Comment:
rename the long option to `--config-file` just for consistency with other projects